aten/src/ATen/cuda/CUDASparseDescriptors.cpp

```diff
@@ -81,7 +81,7 @@ CuSparseDnMatDescriptor::CuSparseDnMatDescriptor(const Tensor& input, int64_t ba
 #endif
 
   auto batch_stride = ndim > 2 && batch_offset >= 0 ? input_strides[ndim - 3] : 0;
-  void* values_ptr = static_cast<char*>(input.data_ptr()) +
+  void* values_ptr = static_cast<char*>(input.mutable_data_ptr()) +
```

Comment:
This looks wrong. Based on the description here I doubt this pointer is actually getting mutated. Generally, you can't trust the underlying libraries to have correct const or not types